website/blog/2020-05-18-rn4mupdadates.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "React Native for macOS"
+author: Kiki Saintonge
+authorURL: http://twitter.com/KikiSaintonge
+description: "We're on React Native core version 0.62 with new community modules, native module support across multiple platforms, and of course our official public release of React Native for macOS."
+is_blog: true
+published_date: May 17, 2020
+---
+
+We're excited to announce our first preview release aligning with [React Native 0.62](https://reactnative.dev/blog/2020/03/26/version-0.62)!
+
+<!--truncate-->
+
+As a preview release, we will try our best not to make breaking changes, but still have a few bumps to sort out before we're ready for release. You can now start trying out the [0.62-preview of React Native for Windows](https://www.npmjs.com/package/react-native-windows/v/0.62.0-preview.6)!

```

Comment:
A similar upgrade for React Native for macOS is [in progress](https://github.com/facebook/react-native/compare/master...microsoft:fb62merge). Stay tuned for the next update!

Reply:
updated with the above text + link.